src/macros/macrorecorder.cpp

```diff
@@ -50,40 +50,40 @@ bool MacroRecorder::checkOrClaimRecording(ChannelHandle* channel) {
 }
 
 bool MacroRecorder::claimRecording() {
-    auto armed = MacroRecordingState::Armed;
-    return m_macroRecordingState.compare_exchange_weak(armed, MacroRecordingState::Recording);
+    auto armed = State::Armed;
+    return m_macroRecordingState.compare_exchange_weak(armed, State::Recording);
 }
```

Comment:
Why is this a method and not part of `checkOrClaimRecording`? Is there any usecase to claim a recording without also verifying that the channel is assigned and matches?

Reply:
Not really, it is nice for testing and I need it multiple times - without it I would have needed to duplicate the code.